src/main/java/com/bgsoftware/superiorskyblock/core/messages/Message.java

```diff
@@ -906,7 +849,7 @@ protected MessagesServiceImpl create() {
     };
 
     @Nullable
-    private static Collection<UUID> noInteractMessages;
+    private static Map<Message, Collection<UUID>> delays;
```

Comment:
You don't need this, you already have Message#delayedMessages (which can be renamed to just delays)